docs/experience/longtasks.md

```diff
@@ -0,0 +1,65 @@
+title: Long Tasks
+description: Gathering information about long tasks during hard and soft page loads
+
+Experience collects information about long tasks that happen during page loads. According to the RAIL Model, tasks that are running for longer than **50ms** and blocking the main thread are considered long and are recorded by Sematext Cloud. 
+
+Long tasks can cause issues such as:
+
+* Clunky animations and scrolling affecting user experience.
```

Comment:
No periods because there are not full sentences

Reply:
Thanks, changed.